certbot/plugins/manual.py

```diff
@@ -43,7 +43,8 @@ class Authenticator(common.Plugin):
 
 {validation}
 
-Before continuing, verify the record is deployed."""
+Before continuing, verify the record is deployed.
+This can be done via cli: watch -n1 dig -t txt {domain}"""
```

Comment:
I'm not sure that all of our users know what "via CLI" means, especially if it's not capitalized to appear like an acronym. It seems that we have quite a few users who aren't used to using CLIs for many tasks, and in some cases who are just beginning to learn them specifically in order to run Certbot!

Maybe we could spell it out a little more explicitly, like

`One way to do this is running the following command at a terminal prompt:`

or

`This can be monitored by running the following command:`

or something similar?

The choice of command itself seems very good to me.

Reply:
@adferrand, should this be wrapped with some kind of logic to prevent it from being displayed on Windows, since I assume the default Windows CLI environment doesn't have `watch` and `dig`? Is there a straightforward alternative that would work on most Windows systems to check whether the TXT record exists?

Reply:
Yes, `nslookup`, that is installed on Windows, and in most Linux environments. Following command will display TXT entries in both system: `nslookup -q=TXT example.com`.